pandas/tests/tools/test_to_datetime.py

```diff
@@ -1086,10 +1086,24 @@ def test_to_datetime_today_now_unicode_bytes(self, arg):
     def test_to_datetime_now_with_format(self, format, expected_ds, string, attribute):
         # https://github.com/pandas-dev/pandas/issues/50359
         result = to_datetime(["2020-01-03 00:00:00Z", string], format=format, utc=True)
-        expected = DatetimeIndex(
+        expected = DatetimeIndex(d
             [expected_ds, getattr(Timestamp, attribute)()], dtype="datetime64[ns, UTC]"
         )
         assert (expected - result).max().total_seconds() < 1
+    
+    
+    #MUDEI AQUI
```

Comment:
thanks for starting this - could you mark this as draft please and ping me when it's ready for review?

Reply:
Thanks for the encouragement, I've already put it as a draft